content/influxdb/v2.4/api-guide/client-libraries/python.md

```diff
@@ -157,12 +158,28 @@ write_api.write(bucket=bucket, org=org, record=p)
 ### Complete example query script
 
 ```python
+import influxdb_client
+from influxdb_client.client.write_api import SYNCHRONOUS
+
+bucket = "<my-bucket>"
+org = "<my-org>"
+token = "<my-token>"
+# Store the URL of your InfluxDB instance
+url="http://localhost:8086"
+
+client = influxdb_client.InfluxDBClient(
+    url=url,
+    token=token,
+    org=org
+)
+
+# Query script
 query_api = client.query_api()
-query = ‘ from(bucket:"my-bucket")\
+query = 'from(bucket:"my-bucket")\
 |> range(start: -10m)\
 |> filter(fn:(r) => r._measurement == "my_measurement")\
 |> filter(fn: (r) => r.location == "Prague")\
```

Comment:
does `(fn:(r)` vs `(fn: (r)` make a difference?

Reply:
Thanks for catching the formatting.  it does not make a difference or cause an error though